test/DisplayBoardTests.cpp

```diff
@@ -35,3 +35,15 @@ TEST_F(DisplayBoardTests, checkDisplayBoard)
 
     ASSERT_EQ(output, compareBoard);
 }
+
+TEST_F(DisplayBoardTests, checkOverloadedOstreamOperator)
+{
+    Board board(1, 1, {{1}}, {{1}});
+    std::string compareBoard = "   +--+\n   |  |\n+--+--+\n|  |??|\n+--+--+\n";
+
+    testing::internal::CaptureStdout();
+    std::cout << board;
```

Comment:
I'm unable to reproduce this feature in main.cpp by simple writing down code as below:
```
#include "Board.hpp"
...
    Board board(1, 1, {{1}}, {{1}});
    std::cout << board;
```

I got error:
`error: no match for ‘operator<<’ (operand types are ‘std::ostream {aka std::basic_ostream<char>}’ and ‘Board’)`

Could you look into it?

Reply:
Ok, I moved declaration to board.hpp, but I still prefer to keep definition in DisplayBoard.cpp

Reply:
What use is to keep it in Displayboard when you can't use it?

Reply:
My intention was to keep definition and declaration in displayBoard.hpp / .cpp because it is completely connected to display functionality. I know that you prefer to include only in main board.hpp, so I moved definition of ostream operator to board.hpp. 
From on site it would be clear to have declaration and definition in the same file.
On the other hand why not to "hide" definition it in DisplayBoard.cpp and do not extend Board.cpp. 
@LordLukin what is your opinion?